schema/spacewalk/susemanager-schema.changes

```diff
@@ -1,3 +1,5 @@
+- Add support for Alibaba Cloud Linux 2
+- Add EPEL 8 GPG key to database so it does not need to be manually accepted
```

Comment:
Accepted for what? AFAIK the database is only for the package providers.

If we are talking about reposync, this is controlled by the build-keys. If we are talking about the clients trusting on keys, we don't do that, we only trust the SUSE Manager client tools for now.

That's is... unless @mcalmer corrects me, but I think this is what he explained to me :-)

Reply:
This is correct.

Reply:
Adding the GPG keyid here in the DB just show "Content Provider" of the package correctly in the UI.

Reply:
Adjusted changelog